kokoro/presubmit.sh

```diff
@@ -10,6 +10,17 @@ export PATH=$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/b
 docker stop $(docker ps --all --quiet) || true
 docker kill $(docker ps --all --quiet) || true
 
+# Restarting Docker for Mac to get around the certificate expiration issue:
+# b/112707824
+# https://github.com/GoogleContainerTools/jib/issues/730#issuecomment-413603874
+# https://github.com/moby/moby/issues/11534
+# TODO: remove this temporary fix once b/112707824 is permanently fixed.
+if [ "${KOKORO_JOB_CLUSTER}" = "MACOS_EXTERNAL" ]; then
+  osascript -e 'quit app "Docker"'
+  open -a Docker
+  while ! docker info > /dev/null 2>&1; do sleep 1; done
```

Comment:
should this timeout not be infinite? Or will kokoro just timeout if this fails?

Reply:
Kokoro has its own timeout, although it is rather large. But I opt for keeping things simple, since it is extremely rare that this fails.

Reply:
does `open -a Docker` return a success/error code?

Reply:
Can't verify myself, but as a core command, I am pretty sure it will return proper codes: https://ss64.com/osx/open.html

Reply:
`open` returns non-zero if the requested application was not found.  The implementation launches the application in the background and then returns, so the wait is still required.

Reply:
sounds good to me.